packages/mobile/src/components/summary-table/SummaryTable.tsx

```diff
@@ -1,21 +1,14 @@
 import type { ReactNode } from 'react'
-import React, { useCallback, useRef, useState } from 'react'
+import React, { useRef, useCallback } from 'react'
 
 import { removeNullable } from '@audius/common'
-import {
-  Animated,
-  LayoutAnimation,
-  TouchableOpacity,
-  UIManager,
-  View
-} from 'react-native'
+import { Animated, View } from 'react-native'
 
 import IconCaretDown from 'app/assets/images/iconCaretDown.svg'
 import { Text } from 'app/components/core'
+import { Expandable, useExpandable } from 'app/components/expandable'
 import { flexRowCentered, makeStyles } from 'app/styles'
 import { useThemeColors, type ThemeColors } from 'app/utils/theme'
-UIManager.setLayoutAnimationEnabledExperimental &&
```

Comment:
don't think we need this bc it's already called [here](https://github.com/AudiusProject/audius-protocol/blob/a7cbaafad99db51ff5dee410bca75fdecdcf4872/packages/mobile/src/app/App.tsx#L49)

Reply:
Cool. I tried to find some docs on whether it needs to be called per component that uses the experimental behavior or just enabled globally. But I didn't find an answer there.